main.go

```diff
@@ -196,7 +248,25 @@ func connectAction(ctx *cli.Context) error {
 	}
 	durations["rhsm"] = time.Since(start)
 
-	/* 2. Register insights-client */
+	/* 2 Show Profile information */
+	// Generate a new http client with mtls configure
+	client := http.NewHTTPClient()
+	if client != nil {
+		// Get the user profile
+		profile, err := getUserProfile(client)
+		if err != nil {
+			errorMessages["rhsm"] = fmt.Errorf("Cannot get the user profile: %w", err)
+		}
+
+		fmt.Printf("\nSystem profile:\n")
+		showUserProfile(profile)
+	}
+
+	fmt.Printf("\nThis system is successfully connected to Red Hat, there may be additional tasks" +
```

Comment:
I don't like the way this sentence is written, and it doesn't include the ● prefix like the rest of the commands. Could we rearrange the output so that all the bullet "connected" messages are printed, and the configuration profile comes last? Something like:

```
● This system is already connected to Red Hat Subscription Management
● Connected to Red Hat Insights
● Activated the rhc daemon

✓ Successfully connected to Red Hat!
🛈 Additional operations may be executing on the system to activate the following features:

FEATURE               ACTIVE  
Remote Configuration  true
Remediations          true
Compliance            true

Run the following command to monitor output: journalctl -u rhcd -f
```

Reply:
BTW: was it discussed with any UX designer?

Shouldn't be this output part of `status` sub-command? I think that it would be useful to know such information anytime (not only after connecting). The `connect` command should only connect the system with some services, but it looks like that calling `GET https://cert.console.redhat.com/api/config-manager/v2/profiles/current` only gather some status information and it doesn't connect system with any new service.

Reply:
I would say this has not being discussed with any UX designer, did you consider it should?

I think this could be a great feature to add to the `status` sub-command.

Reply:
We should include this in the `status` subcommand, yes. This PR is introducing the changes requested by RHIN-501. That RFE doesn't expand the output of such status to the `status` subcommand, but we could certainly do so.